### ISSUE TITLE
The player's hand doesn't change

### ISSUE THREAD

**Issue by notabirb1** [Draylar/identity#99]
I've recently started playing around with this mod, but one thing I noticed is that my player's hand doesn't change, like it does in the forge version by ichun.

**Comment by jaymoooose** [comment]
Idk why it isnt working either... It seems like it works for others

**Comment by TheServerit** [comment]
> Idk why it isnt working either... It seems like it works for others

No, it's not. he's right, Player's arm is his skin hand no matter what creature or animal he is. It's kinda annoying.

**Comment by notabirb1** [comment]
> No, it's not. he's right, Player's arm is his skin hand no matter what creature or animal he is. It's kinda annoying.

That's exactly what I mean. I keep forgetting I'm in morph because of that...

**Comment by TheServerit** [comment]
But morph is not for (the) newest versions, that's why I play in the Identity mod... I hope that someone will fix that (the player's arm and all)...

**Comment by Draylar** [comment]
I honestly have no idea how to even start on this, outside designing a model and texture for every hand or arm in the game (which is a ton of work). I might just disable the hand rendering while you are not a player to start... open to suggestions.

**Comment by James103** [comment]
You may want to take a look at https://github.com/Ladysnake/Requiem/blob/f163fe181d313367e30b1a7a8ceaeccace579f0f/src/main/java/ladysnake/requiem/mixin/client/possession/PlayerEntityRendererMixin.java#L97.

Basically, if you are disguised as a biped entity, render that entity's hand instead of the player's hand, if it exists. Otherwise, don't render anything for the player's hand.

**Comment by TheServerit** [comment]
> I honestly have no idea how to even start on this, outside designing a model and texture for every hand or arm in the game (which is a ton of work). I might just disable the hand rendering while you are not a player to start... open to suggestions.

In Morph mod, they made a hand texture/model for every mob.

**Comment by TheServerit** [comment]
I suppose it was really hard work...
But it's possible...

**Comment by deltanedas** [comment]
having no hand would probably be better to at least convey that you aren't a player.

**Comment by TheServerit** [comment]
Having no hand isn't better

**Comment by deltanedas** [comment]
why would a rabid ender shrew have a player's hand?

**Comment by TheServerit** [comment]
Actually, it really doesn't have, but it's really comfortable to see a hand in front of you.

**Comment by TheServerit** [comment]
The best thing to do is to make a hand texture for every mob.
Like in Morph Mod.
